pyomo/contrib/doe/fim_doe.py

```diff
@@ -332,7 +344,8 @@ def compute_FIM(self, design_values, mode='sequential_finite', FIM_store_name=No
             else:
                 prior_in_use = specified_prior
 
-            FIM_analysis = FIM_result(self.param_name, self.measurement_variables, self.measurement_timeset, self.measurement_extra_index,  prior_FIM=prior_in_use, store_FIM=FIM_store_name, scale_constant_value=self.scale_constant_value)
+            FIM_analysis = FIM_result(self.param_name, self.measurement_variables, self.measurement_timeset, flatten_all_measure=self.flatten_measure_name,
```

Comment:
I'd like to see how `flatten_all_measure` is structured in our discussion today.